### PR TITLE
[IMP] Studio: update automation rules doc for v17

### PR DESCRIPTION
**Pull request by auva-odoo**
task-3745330

### CODE REVIEW

File: content/applications/studio/automated_actions.rst [hunk 1]
```diff
@@ -1,249 +1,297 @@
...
+You can also choose to :guilabel:`Log Calls` to record the payloads received, e.g., to make sure the
+data sent by the source system matches the expected format and content. This also helps identify
+and diagnose any issues that may arise. To access the logs, click the :guilabel:`Logs` smart
+button at the top of the :guilabel:`Automation rules` form.
 
-- To allow the action to be run through the website, tick :guilabel:`Available on the Website` and
-  add a :guilabel:`Website Path`.
+.. note::
+   If you wish to use the webhook's content for a purpose other than to find the record, you can
+   only do so using an :ref:`studio/automated-actions/action/python-code` action.
```
Comment (bouvyd) [inline content/applications/studio/automated_actions.rst:175]: You can add this here: 
> The webhook content is available in the server action context as a `payload` variable (a dictionary that contains the GET parameters or POST JSON body of the incoming request).

File: content/applications/studio/automated_actions.rst [hunk 2]
```diff
@@ -1,249 +1,297 @@
...
 
-The action is used to create a new record on any model.
+Once you have defined the automation rule's :ref:`trigger <studio/automated-actions/trigger>`, click
+:guilabel:`Add an action` to define the action to be executed.
 
-.. note::
-   Selecting a :guilabel:`Target Model` is only required if you want to target another model than
-   the one you are on.
+.. tip::
+   You can define multiple actions for the same trigger/automation rule.
```
Comment (bouvyd) [inline content/applications/studio/automated_actions.rst:186]: they are run in order (e.g. if you have an action to update a record then send an email, the email will use the updates values; if you set the email action before the update action, the email will use the values before the update action is run)

File: content/applications/studio/automated_actions.rst [hunk 3]
```diff
@@ -1,249 +1,297 @@
...
 
-The action is used to schedule a new activity linked to the record. Use the :guilabel:`Activity` tab
-to set it up as usual, but instead of the :guilabel:`Assigned to` field, select an
-:guilabel:`Activity User Type`. Select :guilabel:`Specific User` and add the user under
-:guilabel:`Responsible` if the activity should always be assigned to the same user. To dynamically
-target a user linked to the record, select :guilabel:`Generic User From Record` instead and change
-the :guilabel:`User field name` if necessary.
+.. tip::
+   You can create another automation rule with :ref:`studio/automated-actions/action/update-record`
+   actions to update the fields of the new record if necessary. For example, you can use a
```
Comment (bouvyd) [inline content/applications/studio/automated_actions.rst:269]: great that you mention that!

File: content/applications/studio/automated_actions.rst [hunk 4]
```diff
@@ -1,249 +1,308 @@
...
 
-   .. image:: automated_actions/next-activity-example.png
-      :align: center
-      :alt: Example of a Create Next Activity action
+Execute Code
+------------
 
-.. _studio/automated-actions/action/send-sms:
+This action is used to execute Python code. The available variables are described in the
+:guilabel:`Python Code` tab, which is also used to write your code, or in the :guilabel:`Help` tab.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/studio/automated_actions.rst:290]: Maybe something like:
```suggestion
This action is used to execute Python code. You can write your code into the :guilabel:`Python Code` tab and using these variables:
- env: environment on which the action is triggered
- model: model of the record on which the action is triggered; is a void recordset
- record: record on which the action is triggered; may be void
- records: recordset of all records on which the action is triggered in multi-mode; may be void
- time, datetime, dateutil, timezone: useful Python libraries
- float_compare: utility function to compare floats based on specific precision
- log: log(message, level='info'): logging function to record debug information in ir.logging table
- _logger: _logger.info(message): logger to emit messages in server logs
- UserError: exception class for raising user-facing warning messages
- Command: x2many commands namespace
- To return an action, assign: action = {...}

As a reminder, you can find them as well in the :guilabel:`Python Code` tab or in the :guilabel:`Help` tab.
```

File: content/applications/studio/automated_actions.rst [hunk 5]
```diff
@@ -1,249 +1,308 @@
...
 
-Send SMS Text Message
----------------------
+Send Webhook Notification
+-------------------------
 
-The action is used to send an SMS to a contact linked to the record. To do so, select or create an
-:guilabel:`SMS Template`.
+This action allows to send a POST request with the values of the :guilabel:`Fields` to the URL
```
Comment (LoredanaLrpz) [inline content/applications/studio/automated_actions.rst:295]: stupid question, but what's a POST request? Is it an abbreviation for something? If so, what for? I'd also suggest :abbr:

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
I think I'm done! Overall it's great work: the tips and the examples are very well chosen
I'd just say the last three titles are sad compared to the others that have colors, screenshots, examples, tips.

Big fan of your work as usual x)

File: content/applications/studio/automated_actions.rst [hunk 6]
```diff
@@ -1,249 +1,308 @@
...
-.. _studio/automated-actions/trigger/form-modification:
+You can then define:
 
-Based on Form Modification
---------------------------
+- a :guilabel:`Delay`: Specify the number of minutes, hours, days, or months. To trigger the action
+  before the trigger date, specify a negative number. If you selected the :guilabel:`Based on date
+  field` trigger, you must also select the field to be used to determine the delay.
+- :guilabel:`Extra Conditions`: Click :guilabel:`Add condition`, then specify the conditions to be
+  met to trigger the automation rule. Click :guilabel:`New Rule` to add another condition.
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/studio/automated_actions.rst:108]: It is similar, but different (see the tip on Before Update Domain from line 53). And only triggers in the categories "Timing conditions" or "Custom" offer the possibility to add conditions. I will investigate why!

File: content/applications/studio/automated_actions.rst (hunk 5 above)
Comment (bouvyd) [reply content/applications/studio/automated_actions.rst:295]: my 2 cents:
> Do you think i need to add some sort of explanation

no

> if you're venturing into webhooks, you should know what you're doing

i agree 💯

File: content/applications/studio/automated_actions.rst [hunk 7]
```diff
@@ -1,249 +1,308 @@
...
+Create Record
 -------------
 
-The action is used to subscribe existing contacts to the record.
+This action is used to create a new record on any model.
 
-.. _studio/automated-actions/action/next-activity:
+.. note::
+   Selecting a :guilabel:`Record to Create` is only required if you want to target a model other
+   than the one you are on.
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/studio/automated_actions.rst:273]: I rephrased this a little bit to include your sugegstion and also remove the note (with the new note i added (see your next comment), the text was really sandwiched bewteen 2 notes (and a tip) which looks stupid soooo x)

File: content/applications/studio/automated_actions.rst [hunk 8]
```diff
@@ -1,249 +1,343 @@
...
-<studio/views/general/form>`, even before saving the record. This trigger only works on the user
-interface when a modification is made by a user. If the field is changed through another action and
-not by the user, the action will not run.
+   .. image:: automated_actions/timing-conditions-trigger.png
+      :alt: Example of a Based on date field trigger
 
 .. note::
-   This trigger can only be used with the :ref:`Execute Python Code action
-   <studio/automated-actions/action/python-code>`, so development is required.
+   By default, the scheduler checks for trigger dates every 4 hours, which means that lower
```
Comment (LoredanaLrpz) [inline content/applications/studio/automated_actions.rst:121]:
```suggestion
   By default, the scheduler checks for trigger dates every 4 hours, meaning lower
```

**Review by LoredanaLrpz — APPROVED** [review]
Hey hey, I'm approving because I only have one and a half comments left =D

**Comment by auva-odoo** [comment]
Thank you so much for the thorough review(s) @LoredanaLrpz! 
@robodoo r+

**Pull request closed without merging**